sonar-orchestrator/src/main/java/com/sonar/orchestrator/server/PackagingResolver.java

```diff
@@ -29,12 +29,14 @@
 import java.util.Optional;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import org.jetbrains.annotations.NotNull;
 
 public class PackagingResolver {
 
   private static final String PRIVATE_GROUP_ID = "com.sonarsource.sonarqube";
   private static final String PUBLIC_GROUP_ID = "org.sonarsource.sonarqube";
   private static final Pattern ZIP_VERSION_PATTERN = Pattern.compile("^.*-(\\d++.*)\\.zip$");
+  private static final Pattern BRACKETS_PATTERN = Pattern.compile(".*\\[.*\\]$");
```

Comment:
Not really sure if we need to `.*` or could we just `\s*`

Reply:
I mean the first `.*`

Reply:
I modified the Pattern in order to avoid the backtracking.